it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionControllerISpec.scala

```diff
@@ -290,7 +290,7 @@ class SubscriptionControllerISpec extends BaseControllerISpec with SessionDataMi
         val result = await(controller.submit("addr1")(authenticatedRequest()))
 
         status(result) shouldBe 200
-        checkHtmlResultWithBodyText(result, "This postcode is blocked and cannot be used")
+        checkHtmlResultWithBodyText(result, htmlEscapedMessage("You can't use the postcode you've entered"))
```

Comment:
`htmlEscapedMessage` is intended to take a key in the messages file e.g. `htmlEscapedMessage("error.postcode.blacklisted")`